lib/modules/platform/gitlab/readme.md

```diff
@@ -2,31 +2,29 @@
 
 ## Authentication
 
-First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account.
+First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account, or use a [Group Access Token](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html#bot-users-for-groups).
 
 If you are using a Group access token, the token must have Developer role or higher permissions in order to create issues and merge requests.
 The token must have Maintainer permissions in order to perform Automerge.
+Ensure that you are [rotating the group access token](https://docs.gitlab.com/ee/api/group_access_tokens.html#rotate-a-group-access-token) before it expires in order to keep the same GitLab-generated bot user.
 
-For real runs, give the PAT these scopes:
+For real runs, give the access token these scopes:
 
-- `read_user`
```

Comment:
I'm not sure this should be completely dropped, and I suspect that there should be alternative text added.

For a regular PAT, read_user allows Renovate to retrieve the user's account name and email so that they don't need to configure username and gitAuthor. For a group access token, maybe this isn't possible?

Reply:
I did a test with both PAT (`api`/`read_api`) and group access token (`api`/`read_api`; there is no `read_user` for those) and in both cases Renovate is able to retrieve the user's name and email and sets it as `gitAuthor`. I.e., it seems that the `read_api` and `api` scopes include the ability to read the current user's profile (I could not find any GitLab documentation stating this explicitly).

Reply:
I'd like to double check this via a `curl` command just to be sure. If so then I agree we can remove it